posts/2023-10-20-open-liberty-with-cargo-tracker-ee10.adoc

```diff
@@ -0,0 +1,119 @@
+blog_description: "Experience Jakarta EE and Open Liberty in an industrial environment by running Eclipse Cargo Tracker, an open-source application built to simulate real-world implementations of Jakarta EE."
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= Running Eclipse Cargo Tracker, a Jakarta EE exemplar, with Liberty
+Gilbert Kwan <https://github.com/gkwan-ibm>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+
+Recently, https://eclipse-ee4j.github.io/cargotracker/[Eclipse Cargo Tracker] was updated targeting Jakarta EE 10. The project also supports Open Liberty in addition to Payara and Eclipse GlassFish. Now, you can learn Jakarta EE in an industrial environment by running Cargo Tracker with Open Liberty, all on your own computer.
+
+
+== What is the Cargo Tracker application?
+
+As one of the top Java framework choices for enterprise-level and cloud-native Java applications, Jakarta EE is at the center of projects all over the globe. The Cargo Tracker open-source application was designed to replicate a complex enterprise application by using core Jakarta EE features with professional development standards.
+
+Cargo Tracker simulates an industrially-developed freight tracking web application built with Jakarta EE 10 and domain-driven design principles. The application uses key Jakarta EE features, including Faces, CDI, Enterprise Beans, Bean Validation, RESTful Web Services, and JSON Binding. It also uses Persistence, along with Batch and Messaging.
+
+Cargo Tracker books, tracks, updates, and routes cargo objects. Messaging events are logged to update cargo locations, while persistence ensures that cargo status is not changed or removed when the application is shut down. Each portion of Cargo Tracker represents a step in a real-life shipment process: the customer tracks their cargo using the Public Tracking interface, the shipping company schedules and routes deliveries using the Administration interface, and port handlers log arriving and departing cargo using the Mobile Event Logger. 
+
+image::/img/blog/cargo-tracker-ee10-home.png[Cargo Tracker Home Page,width=50%,align="left"]
+image::/img/blog/cargo-tracker-ee10-map.png[Cargo Tracker Map,width=50%,align="right"]
+
+
+== Run Cargo Tracker with Open Liberty 
+
+Cargo Tracker EE10 supports Java SE 11 and 17. https://developer.ibm.com/languages/java/semeru-runtimes/downloads[IBM Semeru Runtime] is the preferred Java runtime when using Open Liberty. Before you clone the application to your machine, install any required JDKs and ensure that your `JAVA_HOME` system variable is set to a compatible version. To access the repository remotely you'll need to install https://git-scm.com/book/en/v2/Getting-Started-Installing-Git[Git] if you haven't already. Finally, you will need to install https://maven.apache.org/download.cgi[Maven] to build and run the application. 
```

Comment:
Is Cargo Tracker EE10 the official name of this new version. of the app? Is this correct?

Reply:
Would perhaps reword the second sentence to be a little more introductory, as it feels like the audience should know what it is already at the moment.
Maybe something like: "When working with Open Liberty, you can make use of the IBM Semeru Runtime as your chosen Java runtime. This runtime offers performance benefits from deep technology investment in projects such as Eclipse OpenJ9 and is available across a wide variety of hardware and software platforms. To find out more about IBM Semeru Runtime, see our previous blog: Open Liberty and Semeru Runtimes, cloud-native performance that matters[https://openliberty.io/blog/2022/08/19/open-liberty-semeru-performance.html]."

Then I'd start a new line for the part of this paragraph that talks about "before you clone the app....".